core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message.properties

```
@@ -0,0 +1,23 @@
+#The MIT License
+#
+#Copyright (c) 2021, CloudBees, Inc.
+#
+#Permission is hereby granted, free of charge, to any person obtaining a copy
+#of this software and associated documentation files (the "Software"), to deal
+#in the Software without restriction, including without limitation the rights
+#to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#copies of the Software, and to permit persons to whom the Software is
+#furnished to do so, subject to the following conditions:
+#
+#The above copyright notice and this permission notice shall be included in
+#all copies or substantial portions of the Software.
+#
+#THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#THE SOFTWARE.
+
+ExecutorsWarning=Distributed builds are set up so consider disabling builds on the controller. See <a href="https://wiki.jenkins.io/display/JENKINS/Security+implication+of+building+on+master">Security implications of building on master</a>.
```

```suggestion
ExecutorsWarning=Distributed builds are set up so consider disabling builds on the controller. See <a href="https://www.jenkins.io/redirect/building-on-master/" target="_blank" rel="noopener noreferrer">Security implications of building on master</a>.
```

Needs corresponding PR to jenkins.io (can be done at the same time as merging this PR) to introduce the redirect.

See also https://github.com/jenkinsci/jenkins/pull/4910 for when I introduced these attributes.